src/diffusion_kinetics/optimization/forward_model_kinetics.py

```diff
@@ -5,11 +5,13 @@
 import math as math
 
 
-def forwardModelKinetics(kinetics, tsec, TC, geometry:str = "spherical", added_steps:int = 0):  #I NEED TO FIX THE ADDED STEPS PROBLEM HERE STILL
+def forwardModelKinetics(
+    kinetics, tsec, TC, geometry: str = "spherical", added_steps: int = 0
+):  # I NEED TO FIX THE ADDED STEPS PROBLEM HERE STILL
```

Comment:
Is this still something that needs to be addressed? Or can we remove that comment?